ursa/local_manager.py

```diff
@@ -8,12 +8,16 @@ def __init__(self):
         self.graph_dict = {}
         self._graph_config = {}
         self._transaction_id = 0
+        self.versions_to_store = 5
```

Comment:
How about putting this in `self._graph_config`?

`self._graph_config['versions_to_store'] = 5`.

Reply:
Done